examples/oak.ts

```diff
@@ -0,0 +1,71 @@
+import { Application } from 'https://deno.land/x/oak@v10.1.0/mod.ts'
+import { GraphQLHTTP } from '../mod.ts'
+import { makeExecutableSchema } from 'https://deno.land/x/graphql_tools@0.0.2/mod.ts'
+import { gql } from 'https://deno.land/x/graphql_tag@0.0.1/mod.ts'
+
+const typeDefs = gql`
+  type Query {
+    hello: String
+  }
+`
+
+const resolvers = {
+  Query: {
+    hello: (_root: undefined, _args: unknown, ctx: { request: Request }) => {
+      return `Hello World! from ${ctx.request.url}`
+    }
+  }
+}
+
+const schema = makeExecutableSchema({ resolvers, typeDefs })
+
+const resolve = GraphQLHTTP({
+  schema,
+  graphiql: true,
+  context: request => ({ request })
+})
+
+const app = new Application()
+
+app.use((ctx, next) => {
+  if(Deno.args.includes('--cors')) {
+    ctx.response.headers.append('access-control-allow-origin', '*')
```

Comment:
I don't think the deno args part is needed

Reply:
I think it's useful given the normal examples don't provide cors and deno's secure-by-default mentality, but if you want it gone I can remove it